app/ios/Podfile

```
@@ -1,9 +1,37 @@
+use_modular_headers!
+
 # CocoaPods analytics sends network stats synchronously affecting flutter build latency.
 ENV['COCOAPODS_DISABLE_STATS'] = 'true'
 
+# Add script to download XCFrameworks
+def download_xcframework(url, name)
+  require 'shellwords'
+  require 'uri'
+
+  # Validate inputs
+  raise "Invalid URL" unless url =~ URI::regexp
```

rubocop

[warning] `URI::regexp` is obsolete and should not be used. Instead, use `URI::DEFAULT_PARSER.make_regexp`.

(Lint/UriRegexp)